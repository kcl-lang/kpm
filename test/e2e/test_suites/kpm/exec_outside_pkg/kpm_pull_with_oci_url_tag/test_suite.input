kpm pull --tag 1.14 oci://localhost:5002/test/k8s