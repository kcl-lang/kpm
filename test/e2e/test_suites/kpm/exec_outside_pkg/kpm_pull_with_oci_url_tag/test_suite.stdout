start to pull 'oci://localhost:5002/test/k8s' with tag '1.14'
pulling 'test/k8s:1.14' from 'localhost:5002/test/k8s'
pulled 'oci://localhost:5002/test/k8s' in '<workspace>/localhost:5002/test/k8s/1.14' successfully